### ISSUE TITLE
Schedule one participant for August usability testing PILOT session

### ISSUE THREAD

**Issue by jaclinec** [uswds/uswds-site#2217]

**Comment by jaclinec** [comment]
There was a delay in scheduling participants due to a delay in establishing Giftbit giftcard mechanism to pay compensation. We now have it set up, so I plan to begin scheduling participants early next week.

**Comment by jaclinec** [comment]
We are first going to schedule one pilot session next week to see how it goes and what we can learn to make more sessions run smoother.

I reached out to one blind individual who previously showed interest in participating in testing in order to schedule a session next week. I gave available days and times and am just waiting to hear back.

**Comment by jaclinec** [comment]
This is complete - we did the pilot test with a blind person using VO yesterday. I am [documenting lessons learned](https://docs.google.com/document/d/1zuA-QaDiN_nHbk1BSfX4sMcrxH7mLP4rl0lIma29oMg/edit?usp=sharing) and incorporating them into relevant process documents (like our [usability testing workflow doc,](https://docs.google.com/document/d/16IyT85PVC7YW7UBJBtK0AXsMdwbrEHKo1wyg_BUE-Mg/edit?usp=sharing)). I've shared preliminary key takeaways with the team in Slack and will also discuss at our CS UX a11y sync today.
